### PR TITLE
plugins: add install-docker

### PR DESCRIPTION
**Pull request by proppy**
usage:

```
build-debian-cloud gce --codename wheezy --plugin install-docker
```

### CODE REVIEW

File: plugins/install-docker-tasks/files/default/docker [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Docker Upstart and SysVinit configuration file
+
+# Customize location of Docker binary (especially for development testing).
+#DOCKER="/usr/local/bin/docker"
+
+# Use DOCKER_OPTS to modify the daemon startup options.
+#DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4"
```
Comment (jeremyje) [inline plugins/install-docker-tasks/files/default/docker:7]: Commented lines intentional?

File: plugins/install-docker-tasks/files/default/docker [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Docker Upstart and SysVinit configuration file
+
+# Customize location of Docker binary (especially for development testing).
+#DOCKER="/usr/local/bin/docker"
```
Comment (jeremyje) [inline plugins/install-docker-tasks/files/default/docker:4]: Commented?

File: plugins/install-docker-tasks/add-docker-init [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# https://github.com/dotcloud/docker/blob/master/contrib/init/sysvinit-debian/docker
```
Comment (jeremyje) [inline plugins/install-docker-tasks/add-docker-init]: Clarify comment, is this the source of this file?

File: plugins/install-docker-tasks/files/default/docker (hunk 1 above)
Comment (proppy) [inline plugins/install-docker-tasks/files/default/docker:7]: yes, that the default file.

File: plugins/install-docker-tasks/files/default/docker (hunk 2 above)
Comment (proppy) [inline plugins/install-docker-tasks/files/default/docker:4]: yes, that's the default file.

File: plugins/install-docker-tasks/add-docker-init (hunk 3 above)
Comment (proppy) [inline plugins/install-docker-tasks/add-docker-init]: Ack, will add more information.

Comment (proppy) [inline plugins/install-docker-tasks/add-docker-init]: Done.

**Comment by proppy** [comment]
PTAL.

**Pull request closed without merging**